scrapper/lookups/mappings/league_id_mapping.py

```diff
@@ -0,0 +1,120 @@
+# Dictionary mapping league id expected from MLB Stats API -> Human Readable league name
+
+league_id_map = {
+    "103,104": "MLB Major League Baseball",
```

Comment:
@andres-alvarado 

For this mapping, I had to manually create some of the abbreviations, and also hardcoded `103,104` for MLB.

[This](https://statsapi.mlb.com/api/v1/league) is the endpoint I originally pulled the file from.

Note: In the api respomse `"name" : "Caribbean Series",` has MLB abbreviation `CS` instead of `SDC`..

I can add a function to rebuild the`"lookup" dictionaries` from [these endpoints](https://github.com/andres-alvarado/beisbol-analitica/blob/dbc501de5b63def62b6441387eec802552d23f4b/scrapper/lookups/helpers.py#L9-L45), and try to normalize they're formatting.

What are your thoughts? Should I shrink the scope of this PR to just `league_id_map` and stash other code in different branch?